python/hsfs/engine/python.py

```diff
@@ -242,6 +242,40 @@ def read_stream(
     def show(self, sql_query, feature_store, n, online_conn):
         return self.sql(sql_query, feature_store, online_conn, "default", {}).head(n)
 
+    @staticmethod
+    def _convert_column_type(pa_type):
```

Comment:
Would it be possible to re-use this https://github.com/logicalclocks/feature-store-api/blob/master/python/hsfs/engine/python.py#L823?

Reply:
@tdoehmen I have refactored the code.